src/ui/segments/plans/card.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import { useState } from 'react';
+
+import { PlanV2 } from '@/api/sanity/pricing/planv2';
+import PlanBody from '@/ui/segments/plans/body';
+import PlanHeader from '@/ui/segments/plans/header';
+
+export default function PlanCard({ plan }: { plan: PlanV2 }) {
+  const [subscription, setSubscription] = useState<'month' | 'year'>('month');
```

Comment:
Maybe there is a better term for this, e.g. `billingInterval`. The `subscription` reads as the level, e.g. `free` or `pro` in the context of the platform.